### DISCUSSION TITLE
Publish the views

### DISCUSSION THREAD

**Discussion by pablomadariaga** [opcodesio/log-viewer#76] [Ideas]
Excellent package, nothing bad to say for now.

But currently I need to edit the package views, isn't there a way to publish the views?

`php artisan vendor:publish --log-viewer-views`

**Comment by arukompas** [comment]
Hey @pablomadariaga 

View publishing is currently disabled because the Blade views are a mess and I also did not test it properly to make sure that published views can be edited without breaking anything.

Also I'm constantly improving/changing the views, which means more breaking changes for you if you decide to publish them.

For now, I'm not rushing to enable view publishing. Perhaps a bit later in the future!

**Reply by pablomadariaga** [reply]
of course

**Reply by alexxgarci** [reply]
What about this, there's any update?

**Reply by arukompas** [reply]
@alexxgarci hey, the latest version uses Vue on the frontend, and it's still not possible to modify the UI.

The only thing you can do is publish the layout file, in case you need to add some additional css/script tags.
https://log-viewer.opcodes.io/docs/3.x/configuration/access-to-log-viewer#configuring-how-the-front-end-makes-api-requests

**Reply by alexxgarci** [reply]
Thanks @arukompas , maybe i can do something with this workaround
